### PR TITLE
S3 extractor fix

### PR DESCRIPTION
**Pull request by angelosantos4**

### CODE REVIEW

File: nodestream/pipeline/extractors/files.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def read_file_from_handle(self, fp: StringIO) -> Iterable[JsonLikeDocument]:
 
 class JsonFileFormat(SupportedFileFormat, alias=".json"):
     def read_file_from_handle(self, fp: StringIO) -> Iterable[JsonLikeDocument]:
-        return [json.load(fp)]
+        return [json.loads(fp)]
```
Comment (angelosantos4) [inline nodestream/pipeline/extractors/files.py:61]: This class method expects a string as input, so it must use json.loads()

File: nodestream/pipeline/extractors/stores/aws/s3_extractor.py [hunk 2]
```diff
@@ -42,7 +42,7 @@ def __init__(
         self.logger = getLogger(__name__)
 
     def get_object_as_io(self, key: str) -> StringIO:
-        return self.s3_client.get_object(Bucket=self.bucket, Key=key)["Body"]
+        return self.s3_client.get_object(Bucket=self.bucket, Key=key)["Body"].read().decode('utf-8')
```
Comment (angelosantos4) [inline nodestream/pipeline/extractors/stores/aws/s3_extractor.py:45]: self.s3_client.get_object(Bucket=self.bucket, Key=key)["Body"] is a botocore.response.StreamingBody type,
This has a read() method that turns it into a Bytes type, and the decoding turns it into a string.

File: nodestream/pipeline/extractors/files.py (hunk 1 above)
Comment (zprobst) [reply nodestream/pipeline/extractors/files.py:61]: This does not expect a string as input, it expects a `StringIO`.

File: nodestream/pipeline/extractors/stores/aws/s3_extractor.py (hunk 2 above)
Comment (zprobst) [reply nodestream/pipeline/extractors/stores/aws/s3_extractor.py:45]: See comment above, we do not want or need to read to a string.

**Pull request merged**